src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    if (declaration.trim() !== '') {
+      const [key, value] = declaration.split(':');
+
+      styleObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
u can use advance methods for it. Try something like this:
```suggestion
  declarations.filter(declaration => declaration.trim() !== '')
    .reduce((styleObject, declaration) => {
    
      YOUR VERSION
      
      return styleObject;
    }, {});
```